tests/Unit/Expressions/ForClasses/ResideInOneOfTheseNamespacesTest.php

```diff
@@ -35,7 +35,7 @@ public function shouldMatchNamespacesProvider(): array
      * @param mixed $actualFQCN
      * @param mixed $explanation
      */
-    public function test_it_should_match_namespace_and_descendants($expectedNamespace, $actualFQCN, $explanation): void
+    public function test_it_should_match_namespace_and_descendants(string $expectedNamespace, string $actualFQCN, string $explanation): void
```

Comment:
`@param` in phpdoc does not match with the method signature, would it be possibile to remove it?